open-socialism/index.md

```diff
@@ -31,13 +31,13 @@ The problem is it can never be assumed that humans won't be corrupt in any syste
 
 The tribal system dealt with this perfectly in that humans were living close to the brink so they had to work together. Helping one's tribe was helping oneself since the individual is entirely dependent on the tribe for survival. Being corrupt and hoarding food for example would result in an individual being ostracized so corruption was limited. As such self interested behavior was the kind that would also help the tribe as whole.
 
-This problem of having self interested behaviour also help the group is core issue that any system including Communism needs to solve. It is more important in Communism because if everyone works for the state those who run the state have an incredible amount of power. They have the power to rule over the entirety of society and them working for their own self interest and hurting the greater good is much more destructive.
+This problem of having self interested behaviour also help the group is a core issue that any system including Communism needs to solve. It is more important in Communism because if everyone works for the state those who run the state have an incredible amount of power. They have the power to rule over the entirety of society and them working for their own self interest and hurting the greater good is much more destructive.
 
 ## Ok ok, so we should as a first step try and align self interest with the good of society
 
 Humans want to be selfish and get ahead. They want to be prestigious and to have more wealth than others. Ok no problem!
 
-Society will have different levels of leaders and those that get ahead should get more money and prestige. However, the difference in wealth between someone at the top of society should not be hugely different than a member at the bottom of society. It might be something like 5x as much, although this is a knob to be tuned. This much more even spread of wealth will mean (particularly when modern technology is factored in) that the 'lowest' members of society will have an abundance and so should be satisfied.
+Society will have different levels of leaders and those that get ahead should get more money and prestige. However, the difference in wealth between someone at the top of society should not be hugely different than a member at the bottom of society. It might be something like 5x as much, although this is a knob to be tuned. More evenly distributed wealth will mean (particularly when modern technology is factored in) that the 'lowest' members of society will have an abundance and so should be satisfied.
```

Comment:
I wonder if there is another way to say ` that the 'lowest' members `

I had suggested (and removed)  `that the 'lower earning' members ` but this implies some salary, which is not the intent of this statement.

Reply:
Do you think `lowest` is implying something it shouldn't or some other issue with it?

I went with `'lowest'` as I think in a much more even society the lower earners need not have a low amount of wealth.

Reply:
Lowest is good, thanks for clarifying.